src/wp-content/themes/twentyeleven/content-featured.php

```diff
@@ -7,6 +7,9 @@
  * @since Twenty Eleven 1.0
  */
 
+/**
+ * @global string $feature_class Classes for post container of featured posts.
+ */
```

Comment:
Twenty Twenty documents `$wp_query` in the [index.php template](https://github.com/WordPress/wordpress-develop/blob/4a2eb5c23d824b118785607f43a3305499367daa/src/wp-content/themes/twentytwenty/index.php#L27-L31) using a similar short docblock. Is this the best way for templates such as `content-featured.php` or Twenty Seventeen's [content-front-page-panels.php](https://github.com/WordPress/wordpress-develop/pull/5920/files#diff-25eee9ce1ed871f5bc722377d524f079df55f48b75425219bbabeb9c66a02e7e)?